### PR TITLE
check for file before writing it

### PR DESCRIPTION
**Pull request by jdoconnor**

### CODE REVIEW

File: lib/turbo-sprockets/parallel_compiler.rb [hunk 1]
```diff
@@ -34,7 +34,12 @@ def compile_in_parallel(paths)
 
           { 'files' => {}, 'assets' => {} }.tap do |data|
             manifest.find([path]) do |asset|
-              logger.info("Writing #{asset.digest_path}")
+              if File.exist?(asset.digest_path)
```
Comment (camertron) [inline lib/turbo-sprockets/parallel_compiler.rb]: Can we simplify a bit and do:

```ruby
next if File.exist?(asset.digest_path)
logger.info("Writing #{asset.digest_path}")
```

i.e. is there any benefit in writing out the skip message?

**Review by camertron — APPROVED** [review]
Hey cool, this counts toward Hacktoberfest 😁

File: lib/turbo-sprockets/parallel_compiler.rb (hunk 1 above)
Comment (jdoconnor) [reply lib/turbo-sprockets/parallel_compiler.rb]: just debug... i think its safe to skip

**Review by camertron — APPROVED** [review]

**Comment by camertron** [comment]
Published v1.2.3 to Rubygems.

**Pull request merged**